master/custom/factories.py

```diff
@@ -115,6 +116,7 @@ def setup(self, parallel, branch, test_with_PTY=False, **kwargs):
                 command=["make", "pythoninfo"],
                 warnOnFailure=True,
             )
+            env=self.test_environ
```

Comment:
This syntax looks wrong. Do you want to pass it as an argument to ShellCommand?

It seems like env={} is the default: https://github.com/buildbot/buildbot/blob/master/master/buildbot/process/buildstep.py#L1129

Reply:
> This syntax looks wrong. Do you want to pass it as an argument to ShellCommand?

The idea is that by default uses the default and if a class overrides it it uses de override

Reply:
Check http://docs.buildbot.net/latest/manual/configuration/buildsteps.html